This patch is needed to retain the master .so, so that only that filename is
stripped while the related hard links are excluded.
diff --git bin/install_megadrivers.py bin/install_megadrivers.py
index 470137e..7d56def 100644
--- a/bin/install_megadrivers.py
+++ b/bin/install_megadrivers.py
@@ -71,7 +71,7 @@ def main():
             os.chdir(ret)
 
     # Remove meson-created master .so and symlinks
-    os.unlink(master)
+    #os.unlink(master)
     name, ext = os.path.splitext(master)
     while ext != '.so':
         if os.path.lexists(name):
